astrokat/test/test_offline_observe.py

```diff
@@ -0,0 +1,21 @@
+###############################################################################
+# SKA South Africa (http://ska.ac.za/)                                        #
+# Author: cam@ska.ac.za                                                       #
+# Copyright @ 2013 SKA SA. All rights reserved.                               #
```

Comment:
The astrokat repo has a BSD license, so this block is not applicable - please remove.  The LICENSE file is in the root of the project.